CHANGELOG.asciidoc

```diff
@@ -21,7 +21,7 @@ endif::[]
 [[unreleased]]
 === Unreleased
 
-https://github.com/elastic/apm-agent-go/compare/v2.4.6...main[View commits]
+https://github.com/elastic/apm-agent-go/compare/v2.4.7...main[View commits]
```

Comment:
add changelog

Reply:
Perhaps add a note that there was an issue in the module tags for v2.4.6, and that's why there's v2.4.7?